platform/viewer/cypress/integration/common/OHIFUserPreferences.spec.js

```diff
@@ -0,0 +1,571 @@
+
+    // Header should be in English
+    // cy.get('.research-use')
+    //   .scrollIntoView()
+    //   .should('have.text', 'INVESTIGATIONAL USE ONLY');
+    //
+    // Options menu should be in English
+    //   cy.get('[data-cy="options-menu"]').should('have.text', 'Options');
+    //   cy.get('[data-cy="about-item-menu"]')
+    //     .first()
+    //     .should('have.text', 'About');
+    //   cy.get('[data-cy="about-item-menu"]')
+    //     .last()
+    //     .should('have.text', 'Preferences');
+    //  });
+
+    it('checks if Preferences set in Study List Page will be consistent on Viewer Page', function() {
+      // Go go hotkeys tab
+      cy.get('@hotkeysLink')
+        .click()
+        .should('have.class', 'active');
+
+      // Set new hotkey for 'Rotate Right' function
+      cy.get('[data-cy="Rotate Right"]')
+        .click()
+        .type('{shift}Q', { force: true });
+      cy.get('@saveBtn')
+        .scrollIntoView()
+        .click();
+
+      // Open User Preferences modal again
+      cy.openPreferences();
+
+      // Go to General tab
+      cy.get('@generalLink').click();
+
+      // Select Spanish and Save
+      cy.get('#language-select').select('Spanish');
+      cy.get('@saveBtn')
+        .scrollIntoView()
+        .click();
+
+      // Go to Study Viewer page
+      cy.openStudy('MISTER^MR');
+      cy.waitDicomImage();
+      cy.expectMinimumThumbnails(5);
+      cy.initCommonElementsAliases();
+
+      // Check if application is in Spanish
+      // Header should be translated to Spanish
+      cy.get('.research-use')
+        .scrollIntoView()
+        .should('have.text', 'SOLO USO PARA INVESTIGACIÓN');
+
+      // Options menu should be translated
+      cy.get('[data-cy="options-menu"]')
+        .should('have.text', 'Opciones')
+        .click();
+      cy.get('[data-cy="about-item-menu"]')
+        .first()
+        .should('contain.text', 'Acerca de');
+      cy.get('[data-cy="options-menu"]').click(); //Close Options overlay
+
+      //TODO: the following code is blocked by issue 1193: https://github.com/OHIF/Viewers/issues/1193
+      //Once the issue is fixed, the following code should be uncommented
+      // cy.get('[data-cy="about-item-menu"]')
+      //   .last()
+      //   .should('have.text', 'Preferencias');
+
+      // Check if new hotkey is working
+      cy.get('body').type('{shift}Q', { release: false });
+      cy.get('@viewportInfoMidTop').should('contains.text', 'R');
+    });
+  });
+
+  context('Study Viewer Page', function() {
+    before(() => {
+      cy.openStudy('MISTER^MR');
+      cy.waitDicomImage();
+      cy.expectMinimumThumbnails(5);
+    });
+
+    beforeEach(() => {
+      cy.initCommonElementsAliases();
+      cy.resetViewport();
+      cy.wait(200);
+      cy.resetHotkeysPreferences();
+      // Open User Preferences modal
+      cy.openPreferences();
+    });
+
+    it('opens User Preferences modal and verify the displayed information', function() {
```

Comment:
This test description is duplicated on line 14

Reply:
Yes and no. On line 14 I'm checking if everything is displayed as expected when we open User Preferences from the StudyList page. Here we are opening it from Study Viewer page. SO we are checking the same thing but from different origins.